README.md

```diff
@@ -101,6 +101,12 @@ To see full configuration types see [template.lua](./lua/leetcode/config/templat
     ---@type boolean
     logging = true,
 
+    ---@type boolean
+    toggle_console_on_run = false,
+
+    ---@type boolean
+    toggle_console_on_submit = false,
+
```

Comment:
Toggle would mean that when you have the console already opened it would close it upon submitting your solution.
```suggestion
```